eloquent/distribution.yaml

```
@@ -2489,6 +2489,18 @@ repositories:
       url: https://github.com/ros2/rviz.git
       version: eloquent
     status: maintained
+  sick_scan2:
+    release:
+      tags:
+        release: release/eloquent/{package}/{version}
+      url: https://github.com/SICKAG/sick_scan2-release.git
+      version: 0.1.5-1
+    source:
+      type: git
+      url: https://github.com/SICKAG/sick_scan2.git
+      version: https://github.com/SICKAG/sick_scan2/tree/master
```

This should be just the branch reference not the full url.

```suggestion
      version: master
```